src/table/utils/find-visible-rows.ts

```diff
@@ -36,6 +38,52 @@ const findVisibleRows = (rootElement: HTMLElement, totalsPosition: TotalsPositio
   const visibleRowStartIndex = findStartIndex(dataRows, yMin, yMax);
   const visibleRowEndIndex = findEndIndex(dataRows, yMin, yMax);
   return { visibleRowStartIndex, visibleRowEndIndex };
+}
+
+const getFristCellOfRow = (rowIndex: number, cells?: NodeListOf<Element>) => {
+  if (!cells) return undefined;
+  for (let i = 0; i < cells.length; i++) {
+    const strIdx = cells[i].getAttribute('rowindex');
+    if (strIdx && parseInt(strIdx, 10) === rowIndex) return cells[i];
+  }
+  return undefined;
+};
+
+const shouldCellBeIn = (cellRect: DOMRect, min: number, max: number) => {
```

Comment:
I guess you are checking if the entire row should be included our not right? I'd call this something more descriptive like `shouldIncludeRowWithCell` or something

Reply:
Good points! Updated it!